internal/params/json.go

```diff
@@ -14,6 +14,8 @@ import (
 	"github.com/qdm12/ddns-updater/internal/provider"
 	"github.com/qdm12/ddns-updater/internal/provider/constants"
 	"github.com/qdm12/ddns-updater/pkg/publicip/ipversion"
+  "github.com/chmike/domain"
+
```

Comment:
indentation/formatting seems a bit wrong, that's probably why the linter is failing 🤔